framework/SwiftPlot/PlotBorder.swift

```diff
@@ -1,8 +1,8 @@
 public struct PlotBorder{
-    public var topLeft       : Point = Point.zero
-    public var topRight      : Point = Point.zero
-    public var bottomLeft    : Point = Point.zero
-    public var bottomRight   : Point = Point.zero
+    public var topLeft       : Pair<FloatConvertible,FloatConvertible> = zeroPair
```

Comment:
You shouldn't need to specify the Pair type, inference should take care of that for you. I believe you should be able to just use Pair as a type, without the constraints.

Reply:
> You shouldn't need to specify the Pair type, inference should take care of that for you. I believe you should be able to just use Pair as a type, without the constraints.

Yes. I'm sorry. I overlooked this.